main.go

```diff
@@ -2,62 +2,28 @@ package main
 
 import (
 	"assignment-exec/image-builder/builder"
-	"assignment-exec/image-builder/utilities"
 	"flag"
-	"fmt"
-	"github.com/pkg/errors"
 	"log"
 )
 
-var publishImage = flag.Bool("publishImage", false, "Push image to docker hub")
-var assignmentEnv = flag.String("assignmentEnv", "assignment-env.yaml", "Assignment Environment configuration filename")
-var dockerfileName = flag.String("dockerfile", "Dockerfile", "Dockerfile name")
+var publishImage = flag.Bool("publishImage", false, "push image to docker hub")
+var assignmentEnvConfigFilename = flag.String("assignmentEnvConfig", "assignment-env.yaml", "Assignment Environment configuration filename")
+var dockerFilepath = flag.String("dockerfile", "Dockerfile", "Dockerfile name")
```

Comment:
change option to `-dockerfileLoc`. Otherwise, it's misleading.

Reply:
okay